front-end/src/main/kotlin/org/jetbrains/kotlin/spec/entity/test/parameters/testArea/SpecTestsParser.kt

```diff
@@ -0,0 +1,58 @@
+package org.jetbrains.kotlin.spec.entity.test.parameters.testArea
```

Comment:
What is the name of the package? It seems it can be simplified. Or all hierarchy levels actually needed?